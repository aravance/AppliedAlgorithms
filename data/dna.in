AGCCCATCGCCTTGGGCTATATTGATGGGGTATATATACATACCCCGGCCATACTAGCTAGATT
CCCTATGATGTATGTTAGCCACCCGGGGATATATCCTTCTGATGGGAAGGCTTGTAACTTACCA
ACCGGTCGAGTGCGCGGAAGCCGGCCGAA
GTCGTTCGGAATGCCGTTGCTCTGTAAA
GCAGCAGCAA
TTTTTTTTTTTTTT
TTATTTTTTA
GCTCCGA
*
*
