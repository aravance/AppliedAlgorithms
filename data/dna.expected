CCCATGTGTATTTGAGGGGTATATTCATAGGCTTGTAATT:40
GTCGTCGGAAGCCGGCCGAA:20
:0
TA:2
